base/range.jl

```diff
@@ -861,13 +861,15 @@ end
 median(r::Range{<:Real}) = mean(r)
 
 function in(x, r::Range)
+    applicable(-, x, first(r)) || return false
     n = step(r) == 0 ? 1 : round(Integer,(x-first(r))/step(r))+1
```

Comment:
Why? It doesn't define any variable, and makes the function essentially equivalent to `return false` when the condition is false.